npm_and_yarn/spec/fixtures/projects/npm7/packages_name_missing/package-lock.json

```diff
@@ -0,0 +1,163 @@
+{
+    "name": "{{ name }}",
+    "version": "1.0.0",
+    "lockfileVersion": 2,
+    "requires": true,
+    "packages": {
```

Comment:
Could we use a more minimal fixture to test this? Makes it a bit easier to spot what's going on